wiki/models/pluginbase.py

```diff
@@ -26,6 +24,8 @@
 
 """
 
+from django.utils.translation import gettext_lazy as _
```

Comment:
Nit: imports from `django` are ungrouped.

Reply:
I ran `isort` to sort the `imports` in order.

Reply:
I'll run it again on this file. Maybe it was missed somehow.

Reply:
Done.